src/javascripts/helpers/data/showsData.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import firebaseConfig from '../apiKeys';
+
+const dbUrl = firebaseConfig.databaseURL;
+
+const getShows = () => new Promise((resolve, reject) => {

```

Comment:
Do we want to add the event id here as well to ensure that the shows are based on the event_id?

Reply:
Added this in! Thank you!